src/App.js

```diff
@@ -217,7 +290,7 @@ function getQuestion(i) {
     return (
       <div>
         On your own paper please spell out <strong>
-        {chord.toString().replace('M', '').replace('#', '♯').replace(/b/g, '♭')}</strong> in <strong>{clef}</strong> clef. 
+        {chord.toString().replace('M', '').replace('#', '♯').replace(/b/g, '♭')}, { (inversionType === 0) ? 'Root' : util.ordinalSuffix(inversionType) } inversion</strong> in <strong>{clef}</strong> clef.
```

Comment:
I might suggest adding logic to hide "Root Inversion" if it's the only option selected (kinda redundant / potentially confusing). I didn't wanna get too detailed on that decision at this stage.